src/__tests__/ghost.test.js

```diff
@@ -27,7 +27,7 @@ describe('Ghost Chat', () => {
   // jest.setTimeout(2000000)
 
   beforeAll(async () => {
-    ipfs = await utils.initIPFS(11);
+    ipfs = await utils.initIPFS(4);
```

Comment:
Why did you change this? I'm pretty sure that 4 & 5 is being used by another test.

Reply:
Oh it just didn't seem right cause at the end of the test we close 4 and 5. I'll change it back

Reply:
aha! Change the numbers in the end of the test instead then :D